gen3/bin/workon.sh

```diff
@@ -127,5 +153,8 @@ EOM
   fi
 fi
 
-echo "Running: terraform init --backend-config ./backend.tfvars $GEN3_TFSCRIPT_FOLDER/"
-gen3_terraform init --backend-config ./backend.tfvars "$GEN3_TFSCRIPT_FOLDER/"
+tversion=$(checkTerraformModule ${GEN3_TFSCRIPT_FOLDER})
+
+
+echo "Running: terraform${tversion} init --backend-config ./backend.tfvars $GEN3_TFSCRIPT_FOLDER/"
+gen3_terraform${tversion} init --backend-config ./backend.tfvars "$GEN3_TFSCRIPT_FOLDER/"
```

Comment:
what if we did something like `export GEN3_TFORM_VERSION...`, in `gen3 workon`, then have `gen3_terraform` key on that to run the right terraform version, rather than repeating this check multiple places in the code?